architecture/verifier/README.md

```diff
@@ -0,0 +1,83 @@
+# Verifier Refactor Changes Overview
+
+## Configuration
+
+Configuration for each Veraison component is encapsulated by a `ParamStore`
+that contains parameter definitions. A parameter definition specifies the type
+of the parameter (used in validation), and a path where it can be found within
+configuration.
+
+Configuration for a Veraison deployment is handled by a `Config` that gathers
+`ParamStore`s for components used in that deployment. The `Config`uses
+[Viper](https://github.com/spf13/viper) to gather configuration from various
+sources and populate `ParamStore`s associated with it. Parameters are populated
+based on their paths,  and the same value can be used to populate parameters
+inside multiple stores, if they share a path.
+
+<p/>
+<p align="center">
+  <img src="./veraison-config.png"/>
+</p>
+
+`ParamStore`s are created, populated with parameter definitions for their
```

Comment:
`ParamStore`s: We need to correct some indentation here, as it does not come up nicely in the regular text ?

Reply:
Sorry, what do you mean by "indentation" here? If you're talking about about the gap between the "e" and the "s" in "`ParamStore`s" then there is not much we can do about that, as this is how git hub renders this. The alternatives would be
- `ParamStores` -- this implies that the plural form itself an identifier and may lead to confusion.
-  `ParamStore`'s -- the `'` is sometimes used in lieu of formatting to indicate pluralisation of a singular identifier (rather than the plural form itself being an identifier) in plain text. However, given that formatting *is* available, I would argue the apostrophe is not desirable, as it is already overloaded to indicate possessive form and the contraction of "is".
- `ParamStore` instances -- avoids needing to pluralize the identifier at the expense of making the sentence longer and more complex. I would argue this would have a net negative effect on comprehension.

IMO, the current structure is the least bad of the options. However, this is stylistic, and I would be happy to change it to one of the above options?

Reply:
Yes, I meant a rendering issue. Does not come up nicely. Let me ponder over but I agree, we may have to leave it like this, if we do not have other simple option to fix it!